packages/modal/src/Modal/ModalView.tsx

```diff
@@ -75,6 +75,15 @@ const ModalView = React.forwardRef(
 
     useEscapeKey(handleClose, { enabled: open && !isPopoverOpen });
 
+    const focusTrapOptions = initialFocus
+      ? {
+          initialFocus: `#${id} ${initialFocus}`,
```

Comment:
Should we be validating that `initialFocus` is a valid selector string? Or does `FocusTrap` do that for us?

Reply:
I added the types for `focusTrapOptions` which looks likes it checked to make sure its a valid DOM node
```
  export type FocusTargetValue = HTMLElement | SVGElement | string;
  export type FocusTargetValueOrFalse = FocusTargetValue | false;

  /**
   * A DOM node, a selector string (which will be passed to
   * `document.querySelector()` to find the DOM node), or a function that
   * returns a DOM node.
   */
  export type FocusTarget = FocusTargetValue | (() => FocusTargetValue);
```

is that what you mean?